### ISSUE TITLE
The first submission of a fresh setup is always trigger a human captcha

### ISSUE THREAD

**Issue by fpietrosanti** [globaleaks/GlobaLeaks#1400]
Even the first submission of the first installation always trigger a human captcha.

**Comment by evilaliv3** [comment]
i think that this issue is due to the check on the alamrms that in case the disk is full enables also the captcha.

```
Dear Administrator of server XXXXX,

GlobaLeaks technology includes a server status checking component which will alert you in case something requires your attention.

The disk usage is 949MB of 2GB.

Please consider asking your technical support to add more server disk space.

This email will be resent in the next hours if the issue is not resolved.

The level of this alarm is LOW; don’t panic, your system is doing well.
```

**Comment by vecna** [comment]
is the 10% percentage check

**Comment by evilaliv3** [comment]
@vecna: this two things should be totally unrelated and the disk threshold should scatter only the following actions.

1) the captcha should be connected only to rate limiting pre submission and post submission operations.
2) the disk check should only send email to the admin (if first level is exceeded) and then disable the submission (if the hard level is exceeded)

**Comment by vecna** [comment]
* disk status was influencing the alarm level, now not anymore (point 1)
   * is already so, just, with three threshold (low, medium, high).

**Comment by evilaliv3** [comment]
integrated in devel, will be released in 2.60.106 by this morning.
